sdk/loadtestservice/azure-developer-loadtesting/samples/sample.py

```diff
@@ -29,112 +29,151 @@
 
 from azure.developer.loadtesting import LoadTestingClient
 from azure.identity import DefaultAzureCredential
-from azure.core.exceptions import HttpResponseError
 import time
 
+# can be installed using pip install azure-mgmt-loadtestservice
+from azure.mgmt.loadtestservice import LoadTestClient
+from azure.mgmt.loadtestservice.models import LoadTestResource
```

Comment:
I would not include mgmt plane sample code in the sample. I believe management samples should be in this repo: https://github.com/Azure-Samples/azure-samples-python-management/tree/main/samples

cc @msyyc for confirmation

Reply:
> I would not include mgmt plane sample code in the sample. I believe management samples should be in this repo: https://github.com/Azure-Samples/azure-samples-python-management/tree/main/samples
> 
> cc @msyyc for confirmation

@iscai-msft I understand that mgmt package create necessary resource which is needed for data-plane package to test

Reply:
@msyyc that is correct. Management plane creates the resource which is required for any data plane operations. Please let us know if we should keep or remove it.

Reply:
@msyyc management plane should still be separate. All data plane operations do need resource creation first, so we've explicitly decided to split up the samples across the two repos otherwise every data plane sample would include resource creation at the beginning.

Our data plane samples tend to have the information about the already created resource as required environment variables to run the sample. I.e. in text analytics, we require users to input the endpoint and key of the already created resource to run the sample https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/textanalytics/azure-ai-textanalytics/samples/sample_analyze_actions.py#L20

Reply:
Got it. @ninallam  @NiveditJain Please follow @iscai-msft's advice to create the resource before test it.

Reply:
@NiveditJain please make the changes as suggested.